packages/meteor/dynamics_nodejs.js

```diff
@@ -129,54 +99,61 @@ EVp._setNewContextAndGetCurrent = function (value) {
  * @summary Stores the current Meteor environment variables, and wraps the
  * function to run with the environment variables restored. On the server, the
  * function is wrapped within a fiber.
+ *
+ *  This function has two reasons:
+ *  1. Return the function to be executed on the MeteorJS context, having it assinged in the async localstorage.
+ *  2. Better error handling, the error message will be more clear.
  * @locus Anywhere
  * @memberOf Meteor
  * @param {Function} func Function that is wrapped
  * @param {Function} onException
  * @param {Object} _this Optional `this` object against which the original function will be invoked
  * @return {Function} The wrapped function
  */
-Meteor.bindEnvironment = function (func, onException, _this) {
-  Meteor._nodeCodeMustBeInFiber();
-
-  var dynamics = Fiber.current._meteor_dynamics;
-  var boundValues = dynamics ? dynamics.slice() : [];
-
-  if (!onException || typeof(onException) === 'string') {
-    var description = onException || "callback of async function";
-    onException = function (error) {
-      Meteor._debug(
-        "Exception in " + description + ":",
-        error
-      );
+Meteor.bindEnvironment = (func, onException, _this) => {
+  const dynamics = Meteor._getValueFromAslStore(CURRENT_VALUE_KEY_NAME);
+  const currentSlot = Meteor._getValueFromAslStore(SLOT_CALL_KEY);
+
+  if (!onException || typeof onException === 'string') {
+    var description = onException || 'callback of async function';
```

Comment:
Does this need to be `var`?

```suggestion
    const description = onException || 'callback of async function';
```